### PR TITLE
feat(chart-unfurl): Add multi y-axis for chart unfurl

### PR DESCRIPTION
**Pull request by shruthilayaj**
Adding the ability to unfurl multi y-axis charts.

Screen Shot 2021-09-15 at 3 27 05 PM

### CODE REVIEW

File: static/app/chartcuterie/discover.tsx [hunk 1]
```diff
@@ -21,51 +21,118 @@ export const discoverCharts: RenderDescriptor<ChartType>[] = [];
...
+      Object.assign({}, {key}, data.stats[key])
+    );
+    const color = theme.charts.getColorPalette(stats.length - 2);
+
+    const series = stats
+      .sort((a, b) => (a.order ?? 0) - (b.order ?? 0))
+      .map((s, i) =>
+        BarSeries({
+          name: s.key,
+          data: s.data.map(([timestamp, countsForTimestamp]) => ({
```
Comment (edwardgou-sentry) [inline static/app/chartcuterie/discover.tsx:120]: I'm currently using stacked bar graphs on the browser side for multi y axis
image

I'm thinking we should match but I don't have strong opinions on whether stacked or unstacked bar chart is better. Should we stick with stacked for now?
```suggestion
          stack: 'area',
          data: s.data.map(([timestamp, countsForTimestamp]) => ({
```

**Comment by edwardgou-sentry** [comment]
Tests and changes LGTM! Just had a question about stacking bar charts

**Review by edwardgou-sentry — APPROVED** [review]

File: static/app/chartcuterie/discover.tsx (hunk 1 above)
Comment (shruthilayaj) [reply static/app/chartcuterie/discover.tsx:120]: yeah i'll just reflect whatever you're doing in discover!

**Comment by shruthilayaj** [comment]
Screen Shot 2021-09-16 at 11 30 07 AM

**Pull request merged**